cfgov/preprocessed/css/cf-enhancements.less

```diff
@@ -520,6 +520,74 @@ tbody {
 
 
 /* topdoc
+  name: Hide
+  family: cf-core
+  patterns:
+    - name: Utility class
+      markup: |
+        <h1 class="u-hide">
+          Show using JavaScript
+        </h1>
+      codenotes:
+        - .u-hidden;
+      notes:
+        - "Use this class to hide something,
+          perhaps to be shown using JavaScript."
+  tags:
+    - cf-core
+*/
+
+.u-hide {
+  display: none;
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/90108fbd341964c6399abccdc38c62bd91c8406a

Why can't we make the spacing consistent across CSS and JS again?